draft-ietf-dnsop-session-signal.md

```diff
@@ -135,9 +135,9 @@ used to manage session timeouts, termination, and encryption padding.
 All three TLVs defined here are mandatory for all implementations of DSO.
 Further TLVs may be defined in additional specifications.
 
-DSO messages may or may not be acknowledged; this is signaled by providing a
-non-zero message ID for messages that must be acknowledged (request messages) and a zero message
-ID for messages that are not to be acknowledged (unidirectional messages), and is also specified in the definition
+DSO messages may or may not be acknowledged; this is signalled by providing a
```

Comment:
The RFC editor will probably change this - signalled is UK spelling, signaled for US.